### PR TITLE
tdigest overview

### PR DESCRIPTION
**Pull request by AlexanderSaydakov**
The page is not discoverable yet. Let's add to the navigation after checking on the web site.

### CODE REVIEW

File: docs/tdigest/tdigest.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+# t-digest overview
+
+This is an algorithm and data structure for estimating ranks and quantiles of distributions of numerical values.
+
+The implementation in this library is based on the MergingDigest in following [reference implementation](https://github.com/tdunning/t-digest).
+
+The implementation in this library has a few differences from the reference implementation:
+
+* merge does not modify the input
+* different serialization similar to other sketches in this library, reading the reference implementation format is supported
```
Comment (jmalkin) [inline docs/tdigest/tdigest.md]: I think remove `different` and add a transition:
serialization similar to other sketches in this library, although reading the reference implementation format is supported

File: docs/tdigest/tdigest.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+The library contains a few different quantile sketches for estimating distributions (ranks and quantiles). All other quantile skeches can handle arbitrary comparable types and always retain a small set of items from the input stream. All queries that return approximations in the input domain return one of the retained items from the input. t-digest is different: it works on numeric data only (floating point types), retains and returns values not necessarilly seen in the input (interpolated).
+
+The closest alternative to t-digest in this library is [REQ sketch](https://datasketches.apache.org/docs/REQ/ReqSketch.html). It prioritizes one chosen side of the rank domain: either low rank accuracy or high rank accuracy. t-digest (in this implementation) prioritizes both ends of the rank domain and has lower accuracy towards the middle of the rank domain (median).
+
+The more input values t-digest observes the more it tends to be biased (tends to underestimate low ranks and overestimate high ranks):
```
Comment (jmalkin) [inline docs/tdigest/tdigest.md]: I think it's worth noting that it does that while still doing very well at the actual extremes

**Review by jmalkin — APPROVED** [review]
Request a few minor changes but approving since they're small enough I don't need to re-review

File: docs/tdigest/tdigest.md (hunk 1 above)
Comment (AlexanderSaydakov) [reply docs/tdigest/tdigest.md]: agree

File: docs/tdigest/tdigest.md (hunk 2 above)
Comment (AlexanderSaydakov) [reply docs/tdigest/tdigest.md]: Perhaps, we could soften this statement a bit. How about:
Measurements show that t-digest is slightly biased (tends to underestimate low ranks and overestimate high ranks), while still doing very well close to the extremes. The effect seems to be more pronounced with more input values.

Comment (jmalkin) [reply docs/tdigest/tdigest.md]: That works for me

Comment (AlexanderSaydakov) [reply docs/tdigest/tdigest.md]: implemented

**Pull request merged**